vello_encoding/src/math.rs

```diff
@@ -122,7 +123,8 @@ pub(crate) fn f32_to_f16(val: f32) -> u16 {
     output | (sign >> 16) as u16
 }
 
-/// Convertes a 16-bit precision IEEE-754 binary16 float to a f32.
+/// Converts a 16-bit precision IEEE-754 binary16 float to a `f32`.
+///
 /// This implementation was adapted from Fabian Giesen's `half_to_float()`
 /// function which can be found at <https://gist.github.com/rygorous/2156668#file-gistfile1-cpp-L574>
 pub fn f16_to_f32(bits: u16) -> f32 {
```

Comment:
The final three items in this module paint a funny story:
- `#[allow(dead_code)] fn point_to_f32`
- `pub(crate) fn f32_to_f16`
- `pub fn f16_to_f32` (i.e. the inverse of the `pub(crate)` item above)

Reply:
And it is a public mod so there's a chance that skia is actually. using something. Who knows!

Reply:
And that `allow(dead_code)` should be a `full` feature gate.